crawler/centre_file_state.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum, auto
```

Comment:
Nice, I wanted these to be separate files 👍

Reply:
I wanted to get the Centre and CentreFile in separate files too because the `file_processing.py` file is getting out of hand but have deferred that to a larger refactor.

Reply:
yep sounds good